SpringProject/ElasticSearch/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '2.0'
+
+services:
+  mysql:
+    image: mysql:5.7
```

Comment:
Why are you using an older version of mysql? 8.0 is the latest version

Reply:
Good question, I all honesty I am not sure there is a reason! I have updated this to say mysql:8.0 and it still runs so I guess its okay?

Reply:
Yes probably although I suspect Mendeley infra can't handle 8.0 so check where this is being built and deployed (if it is)

Reply:
We are building from scratch so I think it should be fine with 8.0 thanks!